# **About the notebook, data, and story:** 
This notebook contains a CSV derived from Reporters Without Borders' 2026 Press Freedom Index, which I use to analyze press freedom scores across the Americas, a region RSF has shown great concern for. In order to evaluate the factors contributing to the region's declining press freedom scores and the extent to which the region and its countries have worsened, I have addressed the following questions:

**1)** How do the Americas compare with other RSF regions in overall press freedom scores in 2026? 

**2)** How have the Americas press freedom scores evolved (gone up or down, and by how much) per country compared to 2025? (RSF noted the region has fallen 14 points since 2022)

**3)** Which countries in the Americas have the highest vs the weakest press freedom, in turn impacting the region's global position? (In other words, which countries have scores that have the most sway when it comes to the region's average -  differs from Q2 as it focuses on overall score by country, not by how much each has fallen in 1 year reflecting trajectory)

**4)** Which Press Freedom indicators are weakest across the region? / Which of RSF's five indicators has the lowest average score across the Americas (helps reveal which aspect of press freedom has weakened the most in the region - zooms in more than score)? 

## **Step 1:** Check to see where the CSV is before proceeding. 

In [1]:
import os

print(os.getcwd())

#imported this to see where it was finding the CSV. Originally had it in a diff folder and it said the CSV didn't exist.

/Users/avagough/Desktop/LEDE/LedeProj2


## **Step 2:** Import pandas library and read the CSV. Look at comments in the cell below to understand why I skipped row 1. Then proceed with checking out the CSV (rows, columns, etc.)

In [2]:
import pandas as pd

df = pd.read_csv("RSF_2026.csv", skiprows=1) 

#had to skiprows=1 b/c it appears my CSV had an extra first line showing '2026 (1)' so pandas thought it was the first row
#having that made it difficult to make it into columns, which originally replied with 'Index(['2026 (1)'], dtype='str')'

In [3]:
df.head(10) #wanted to see that my df matched the csv and list on the RSF webpage. 

,ISO,Score 2026,Rank,Political Context,Rank_Pol,Economic Context,Rank_Eco,Legal Context,Rank_Leg,Social Context,...,Country_EN,Country_ES,Country_PT,Country_AR,Country_FA,Year (N),Rank N-1,Rank evolution,Score N-1,Score evolution
0,FIN,"86,22",6,"89,16",8,"78,68",5,"85,62",10,"83,59",...,Finland,Finlandia,Finlândia,??????,??????,2026,5,-1,"87,18","-0,96"
1,EST,"88,54",3,"90,29",4,"75,33",8,"89,25",5,"92,33",...,Estonia,Estonia,Estônia,???????,??????,2026,2,-1,"89,46","-0,92"
2,NLD,"88,92",2,"89,31",7,"85,38",2,"90,68",2,"87,7",...,Netherlands,Países Bajos,Países Baixos,??????,????,2026,3,1,"88,64","0,28"
3,SWE,"87,61",5,"89,61",6,"81,67",3,"88,82",6,"84,53",...,Sweden,Suecia,Suécia,??????,????,2026,4,-1,"88,13","-0,52"
4,LTU,"81,34",15,"74,05",24,"68,03",14,"86,29",8,"85,82",...,Lithuania,Lituania,Lituânia,????????,???????,2026,14,-1,"82,27","-0,93"
5,DNK,"88,47",4,"91,79",2,"79,51",4,"86,56",7,"88,79",...,Denmark,Dinamarca,Dinamarca,????????,???????,2026,6,2,"86,93","1,54"
6,IRL,"85,93",7,"90,5",3,"76,59",6,"80,82",25,"86,2",...,Ireland,Irlanda,Irlanda,???????,??????,2026,7,0,"86,92","-0,99"
7,PRT,"83,71",10,"86,76",9,"62,25",26,"85,86",9,"89,29",...,Portugal,Portugal,Portugal,????????,??????,2026,8,-2,"84,26","-0,55"
8,CHE,"84,83",8,"89,65",5,"70,67",11,"82,74",18,"85,86",...,Switzerland,Suiza,Suíça,??????,?????,2026,9,1,"83,98","0,85"
9,CZE,"83,01",11,"76,25",20,"66,46",17,"89,53",4,"89,31",...,Czechia,Chequia,Chéquia,??????,??,2026,10,-1,"83,96","-0,95"


In [4]:
df.info() #shows what is inside CSV and types (str & int were dealt with later.) 

<class 'pandas.DataFrame'>
RangeIndex: 180 entries, 0 to 179
Data columns (total 25 columns):
 #   Column             Non-Null Count  Dtype
---  ------             --------------  -----
 0   ISO                180 non-null    str  
 1   Score 2026         180 non-null    str  
 2   Rank               180 non-null    int64
 3   Political Context  180 non-null    str  
 4   Rank_Pol           180 non-null    int64
 5   Economic Context   180 non-null    str  
 6   Rank_Eco           180 non-null    int64
 7   Legal Context      180 non-null    str  
 8   Rank_Leg           180 non-null    int64
 9   Social Context     180 non-null    str  
 10  Rank_Soc           180 non-null    int64
 11  Safety             180 non-null    str  
 12  Rank_Saf           180 non-null    int64
 13  Zone               180 non-null    str  
 14  Country_FR         180 non-null    str  
 15  Country_EN         180 non-null    str  
 16  Country_ES         180 non-null    str  
 17  Country_PT         180 non-

In [5]:
df.shape #shows my data has 180 rows (aka countries) and 25 columns (aka categories).

(180, 25)

In [6]:
df.columns #Gives a list of the 25 columns. 

Index(['ISO', 'Score 2026', 'Rank', 'Political Context', 'Rank_Pol',
       'Economic Context', 'Rank_Eco', 'Legal Context', 'Rank_Leg',
       'Social Context', 'Rank_Soc', 'Safety', 'Rank_Saf', 'Zone',
       'Country_FR', 'Country_EN', 'Country_ES', 'Country_PT', 'Country_AR',
       'Country_FA', 'Year (N)', 'Rank N-1', 'Rank evolution', 'Score N-1',
       'Score evolution'],
      dtype='str')

In [7]:
df.columns.tolist() #Gives me a list of the 25 columns. Easier to read than above. 

['ISO',
 'Score 2026',
 'Rank',
 'Political Context',
 'Rank_Pol',
 'Economic Context',
 'Rank_Eco',
 'Legal Context',
 'Rank_Leg',
 'Social Context',
 'Rank_Soc',
 'Safety',
 'Rank_Saf',
 'Zone',
 'Country_FR',
 'Country_EN',
 'Country_ES',
 'Country_PT',
 'Country_AR',
 'Country_FA',
 'Year (N)',
 'Rank N-1',
 'Rank evolution',
 'Score N-1',
 'Score evolution']

## **Step 3:** Find the "Zones" aka the regions in order to extract further information from the Americas and to compare it with other regions and the countries within the Americas later on. Will be useful for visuals and analysis.

In [8]:
df["Zone"].unique() #gives me the specific "Zones," so I can pull countries from the Americas.

<StringArray>
['UE Balkans', 'Asie-Pacifique', 'Amériques', 'Afrique', 'EEAC', 'MENA']
Length: 6, dtype: str

## **Step 4:** Filter through the Americas to get a list of the countries in this "Zone." Make sure everything looks right.

In [9]:
Americas = df[df["Zone"] == "Amériques"] #this line grabs the countries in the Americas.

In [10]:
Americas.shape #this shows us the number of countries in the Americas. The 25 is the columns.

(28, 25)

In [11]:
Americas.head(28) 
#Important to note that the numbers to the far left are just the order in the original list in CSV, not their score or rank.
#For example, Trinidad and Tobago is not ranked 18 out of 180 countries, it's just the 18th country in the list. 

,ISO,Score 2026,Rank,Political Context,Rank_Pol,Economic Context,Rank_Eco,Legal Context,Rank_Leg,Social Context,...,Country_EN,Country_ES,Country_PT,Country_AR,Country_FA,Year (N),Rank N-1,Rank evolution,Score N-1,Score evolution
18,TTO,"74,7",32,"64,6",39,"62,4",25,"77,21",35,"78,79",...,Trinidad and Tobago,Trinidad y Tobago,Trinidad e Tobago,???????? ???????,????? ? ??????,2026,19,-13,"79,71","-5,01"
20,CAN,"78,76",20,"82,75",12,"63,7",23,"80,54",27,"78,29",...,Canada,Canadá,Canadá,????,??????,2026,21,1,"78,75","0,01"
25,JAM,"75,87",26,"64,76",38,"67,04",15,"72,62",47,"81,85",...,Jamaica,Jamaica,Jamaica,???????,????????,2026,26,0,"75,83","0,04"
31,SUR,"73,2",34,"62,33",42,"66,71",16,"68,06",60,"78,98",...,Suriname,Surinam,Suriname,???????,???????,2026,32,-2,"74,49","-1,29"
35,CRI,"72,35",38,"50,73",71,"58,11",37,"83,34",15,"77,41",...,Costa Rica,Costa Rica,Costa Rica,?????????,?????????,2026,36,-2,"73,09","-0,74"
42,DOM,"69,73",44,"56,35",56,"51,41",51,"84,39",12,"81,5",...,Dominican Republic,República Dominicana,República Dominicana,??????? ???????????,?????? ???????,2026,43,-1,"69,87","-0,14"
46,BLZ,"61,66",66,"49,92",74,"46,23",64,"60,2",84,"68,32",...,Belize,Belice,Belize,????,????,2026,47,-19,"68,32","-6,66"
47,CSS,"64,6",57,"55,01",58,"51,32",52,"58,88",90,"67,06",...,OECS,OECO,OECO,????????,????? ??????? ????,2026,48,-9,"68,08","-3,48"
52,PAN,"62,14",65,"50,44",73,"46,67",63,"62,47",78,"63,41",...,Panama,Panamá,Panamá,????,??????,2026,53,-12,"66,75","-4,61"
56,USA,"62,61",64,"53,59",62,"55,05",41,"70,47",52,"64,27",...,United States,Estados Unidos,Estados Unidos,???????? ???????,?????? ????? ??????,2026,57,-7,"65,487","-2,87"


In [12]:
Americas["Country_EN"] #lists the countries in the Americas in English (RSF has them in other lang too.)

18     Trinidad and Tobago
20                  Canada
25                 Jamaica
31                Suriname
35              Costa Rica
42      Dominican Republic
46                  Belize
47                    OECS
52                  Panama
56           United States
58                 Uruguay
62                  Brazil
68                   Chile
72                  Guyana
83                Paraguay
86               Argentina
92                 Bolivia
93                 Ecuador
110                  Haiti
114               Colombia
123                 Mexico
129                   Peru
134            El Salvador
137              Guatemala
141               Honduras
159              Venezuela
164                   Cuba
171              Nicaragua
Name: Country_EN, dtype: str

In [13]:
#Noticed in the cell for .head() that there are commas instead of decimals for numbers. Used the code below to see why.
#The response below shows that some scores are strings instead of integers, which I initially missed when doing df.info().
Americas.info()

<class 'pandas.DataFrame'>
Index: 28 entries, 18 to 171
Data columns (total 25 columns):
 #   Column             Non-Null Count  Dtype
---  ------             --------------  -----
 0   ISO                28 non-null     str  
 1   Score 2026         28 non-null     str  
 2   Rank               28 non-null     int64
 3   Political Context  28 non-null     str  
 4   Rank_Pol           28 non-null     int64
 5   Economic Context   28 non-null     str  
 6   Rank_Eco           28 non-null     int64
 7   Legal Context      28 non-null     str  
 8   Rank_Leg           28 non-null     int64
 9   Social Context     28 non-null     str  
 10  Rank_Soc           28 non-null     int64
 11  Safety             28 non-null     str  
 12  Rank_Saf           28 non-null     int64
 13  Zone               28 non-null     str  
 14  Country_FR         28 non-null     str  
 15  Country_EN         28 non-null     str  
 16  Country_ES         28 non-null     str  
 17  Country_PT         28 non-null  

In [14]:
#Given the above issue, I needed to convert the strings into integers to remove decimals with code below.

score_columns = [
    "Score 2026",
    "Political Context",
    "Economic Context",
    "Legal Context",
    "Social Context",
    "Safety",
    "Score N-1",
    "Score evolution"
]

for col in score_columns:
    Americas[col] = (
        Americas[col]
        .str.replace(",", ".", regex=False)
        .astype(float)
    )

In [15]:
#This cell / code is to now check that the above code worked when attempting to fix the comma issue.
#We see it worked because instead of 'str' it now shows 'float64' which indicates a number with a decimal. 
Americas.info()

<class 'pandas.DataFrame'>
Index: 28 entries, 18 to 171
Data columns (total 25 columns):
 #   Column             Non-Null Count  Dtype  
---  ------             --------------  -----  
 0   ISO                28 non-null     str    
 1   Score 2026         28 non-null     float64
 2   Rank               28 non-null     int64  
 3   Political Context  28 non-null     float64
 4   Rank_Pol           28 non-null     int64  
 5   Economic Context   28 non-null     float64
 6   Rank_Eco           28 non-null     int64  
 7   Legal Context      28 non-null     float64
 8   Rank_Leg           28 non-null     int64  
 9   Social Context     28 non-null     float64
 10  Rank_Soc           28 non-null     int64  
 11  Safety             28 non-null     float64
 12  Rank_Saf           28 non-null     int64  
 13  Zone               28 non-null     str    
 14  Country_FR         28 non-null     str    
 15  Country_EN         28 non-null     str    
 16  Country_ES         28 non-null     str    

## **Step 5:** Find the average press freedom score by RSF region. This will be used to compare the Americas to the other regions we found earlier in **Step 3** and see and how their standings evolved from 2025 (several cells below). The information from this will also be used to generate a chart to answer - **1) How do the Americas compare with other RSF regions in overall press freedom scores in 2026?** 

In [16]:
#score for all regions and 5 indicators, not just for americas or last df. 
score_columns = [
    "Score 2026",
    "Political Context",
    "Economic Context",
    "Legal Context",
    "Social Context",
    "Safety",
    "Score N-1",
    "Score evolution"
]

for col in score_columns:
    df[col] = (
        df[col]
        .astype(str)
        .str.replace(",", ".", regex=False)
        .astype(float)
    )

In [17]:
df.info()

<class 'pandas.DataFrame'>
RangeIndex: 180 entries, 0 to 179
Data columns (total 25 columns):
 #   Column             Non-Null Count  Dtype  
---  ------             --------------  -----  
 0   ISO                180 non-null    str    
 1   Score 2026         180 non-null    float64
 2   Rank               180 non-null    int64  
 3   Political Context  180 non-null    float64
 4   Rank_Pol           180 non-null    int64  
 5   Economic Context   180 non-null    float64
 6   Rank_Eco           180 non-null    int64  
 7   Legal Context      180 non-null    float64
 8   Rank_Leg           180 non-null    int64  
 9   Social Context     180 non-null    float64
 10  Rank_Soc           180 non-null    int64  
 11  Safety             180 non-null    float64
 12  Rank_Saf           180 non-null    int64  
 13  Zone               180 non-null    str    
 14  Country_FR         180 non-null    str    
 15  Country_EN         180 non-null    str    
 16  Country_ES         180 non-null    st

### **Code below gives numbers for Chart 1:** Average Press Freedom Score by RSF Region (2026)

In [18]:
#the above checked for floats. 
#this cell will now calculate regional avg. for CHART 1.

region_scores = df.groupby("Zone")["Score 2026"].mean().sort_values(ascending=False)

region_scores

Zone
UE Balkans        73.730750
Amériques         55.347143
Afrique           53.072917
Asie-Pacifique    47.717813
EEAC              39.316923
MENA              36.333158
Name: Score 2026, dtype: float64

In [19]:
#Convert it into a DF to change all region names to be in English. Americas in French this time even after last change b/c of old DF.

region_scores_df = region_scores.reset_index()

region_scores_df["Zone"] = region_scores_df["Zone"].replace({
    "Amériques": "Americas",
    "Asie-Pacifique": "Asia-Pacific",
    "Afrique": "Africa",
    "UE Balkans": "Europe/Balkans"
})

NOTE: RSF calls Eastern Europe - Central Asia EECA in English but EEAC in French,
Which i forgot to adjust, so in the region ranking it says EEAC but this error
is fixed in my visuals and the code remains accurate

In [20]:
#check that the above worked

region_scores_df

,Zone,Score 2026
0,Europe/Balkans,73.730750
1,Americas,55.347143
2,Africa,53.072917
3,Asia-Pacific,47.717813
4,EEAC,39.316923
5,MENA,36.333158


In [41]:
#added this a bit later after realizing/remembering that the CSV includes scores from
#2025, allowing me to find the average for regions based on their coutries' scores in 2025.
region_change = df.groupby("Zone")[["Score N-1", "Score 2026", "Score evolution"]].mean().round(2)
region_change = region_change.sort_values("Score evolution") 

region_change_df = region_change.reset_index() #renamed again bc new df name.
region_change_df["Zone"] = region_change_df["Zone"].replace({
    "Amériques": "Americas",
    "Asie-Pacifique": "Asia-Pacific",
    "Afrique": "Africa",
    "UE Balkans": "Europe/Balkans"
})
region_change_df

,Zone,Score N-1,Score 2026,Score evolution
0,EEAC,40.80,39.32,-1.48
1,Americas,56.43,55.35,-1.08
2,Africa,53.55,53.07,-0.48
3,Europe/Balkans,74.03,73.73,-0.30
4,Asia-Pacific,47.70,47.72,0.02
5,MENA,35.87,36.33,0.46


### **Analysis 1:** 

The Americas (55.35) are the second highest performing region of the 6 evaluated by RSF in 2026. While not the lowest region on the totem pole, the Americas still fall 18.38 points behind Europe/Balkans (73.8), and narrowly miss 3rd place, a spot held by Africa, by 2.27 points, making them more comparable in terms of press freedom. 

In 2025, the Americas average score was 56.43, so its overall average dropped by -1.08. This makes it the region with the second-worst decline from 2025, with EEAC (prev avg score of 40.80) dropping by -1.48. Surprisingly, MENA (prev 35.87) actually improved by 0.46 points.

In RSF's Americas 2026 review (https://rsf.org/en/classement/2026/americas), the organization said the region's press freedom index has dropped 14 points since 2022, and compared its downward trajectory to trends in Eastern Europe and Central Asia (EECA) (39.32) and the Middle East and North Africa (MENA)(36.33), which are two of the harshest regions for journalists to operate and the two lowest ranked regions in 2026. As shown in the above chart, the Americas remain about 16 points ahead of EEAC and 19 points ahead MENA. 




## **Step 6:** Find the changes in press freedom scores (not overall shifts in rank) in the Americas (2025–2026 average). This will assist with the **MAP visual** and answers - **2) How has the Americas press freedom scores evolved per country compared to 2025?**

In [21]:
#use the old Americas DF for numbers to analyze/use for map
#score evolution = amount changed 
#country_en = country names

#don't forget that the number to the far left is irrelevant - pay attention to far right number and will see change and by how much.

#Remember OECS is counted as 1 country but is cluster of islands.

Americas["Score evolution"].dtype

Americas[["Country_EN", "Score evolution"]].sort_values("Score evolution")

,Country_EN,Score evolution
93,Ecuador,-9.39
46,Belize,-6.66
129,Peru,-5.02
18,Trinidad and Tobago,-5.01
52,Panama,-4.61
86,Argentina,-3.70
47,OECS,-3.48
56,United States,-2.87
134,El Salvador,-2.31
83,Paraguay,-2.17


#### **Note - MAP:** Code below fixes issues with how RSF wrote out the country names with how Datawrapper wanted them to be. This is for the Map which will reflect the growth and decline each country in the Americas experienced between RSF's 2025 Press Freedom Index report and theirs from 2026.

In [22]:
#make this into csv for a map below

map_data = Americas[["Country_EN", "Score evolution"]]

In [23]:
map_data["Country_EN"] = (
    map_data["Country_EN"]
    .str.strip()                    
    .str.replace("\u00A0", " ", regex=False) 
)

map_data["Country_EN"] = map_data["Country_EN"].replace({
    "Guatemala": "Guatemala",
    "Bolivia": "Plurinational State of Bolivia",
    "Venezuela": "Bolivarian Republic of Venezuela",
    "United States": "United States of America" 
})

#When the CSV for this map/code was put into datawrapper, it did not recognize the names
#for the countries listed above as they were written in RSF's csv, so I had to match them.

In [24]:
map_data[
    map_data["Country_EN"].str.contains(
        "United|Bolivia|Venezuela|OECS|Guatemala",
        case=False,
        na=False
    )
]

,Country_EN,Score evolution
47,OECS,-3.48
56,United States of America,-2.87
92,Plurinational State of Bolivia,0.16
137,Guatemala,2.89
159,Bolivarian Republic of Venezuela,1.27


In [25]:
map_data = map_data[map_data["Country_EN"] != "OECS"]

### **Analysis 2:** 

Of the 28 countries in the Americas (which includes OECS - a unit comprised of multiple islands marked as 1 country by RSF) Ecuador's press freedom score has gone down the most in the last calendar year (-9.39), while the Dominican Republic (-0.14) has dropped the least out of those whose scores have fallen. In total, 17 of 28 countries have fallen (find percentage to include). 

Out of the countries whose scores have increased, Uruguay has made the most progress (3.54) followed by Cuba (3.19). Canada has made the least progress, increasing in press freedom by only 0.01 points since 2025.  


## **Step 7:** Find press freedom scores across countries in the Americas to compare to one another, not to the world. This is for our **Chart 2** (visual 3). This addresses **- 3) Which countries in the Americas have the highest vs the weakest press freedom, in turn impacting the region's global position? (In other words, which countries have scores that have the most sway when it comes to the region's average -  differs from Q2 as it focuses on overall score by country, not by how much each has fallen in 1 year reflecting trajectory)** and Q2 a bit as well, but is different from the map (which specifically shows by how much a country's index has gone up or down in last year) as it shows score for each country compared to one another. 

In [34]:
chart2_data = Americas[["Country_EN", "Score 2026"]].copy()

In [35]:
chart2_data.sort_values("Score 2026", ascending=False) 
#ignore the numbers in bold on the far left again - they are still signalling where they 
#popped up on the general list of countries, it isn't their score or rank. 

,Country_EN,Score 2026
20,Canada,78.76
25,Jamaica,75.87
18,Trinidad and Tobago,74.70
31,Suriname,73.20
35,Costa Rica,72.35
42,Dominican Republic,69.73
58,Uruguay,68.72
62,Brazil,66.37
47,OECS,64.60
56,United States,62.61


### **Analysis 3:** 

While there have been some instances of progression in the Americas (refer to map), this does mean they all have strong press freedom, or that all those who have improved have strong media environments/press freedom scores. In fact, a majority of the Americas, both those that have improved and declined in terms of press freedom, generally fall low on RSF's press freedom indicator rating based on their overall scores and not their growth/decline in the last year, which is divided into the following categories: 

**The organization divides its scores into the following categories:** 

Very serious: 0-40 
Difficult: 40 - 55
Problematic: 55 - 70
Satisfactory: 70 - 85
Good: 85-100 

**VERY SERIOUS:** Five countries, including Cuba, Venezuela, and Nicaragua,  which showed some minor impovement this year, still remain in the "very serious" category.

**DIFFICULT:** Nine countries are listed as "difficult" and of those 9 only  Bolivia, Colombia, Guatemala, and Honduras were said to improve in the last year. 

**PROBLEMATIC:** Another 9 countries were classified as problematic, including the US, which RSF has singled out for its significant shift in press freedom (fallen 7 places in overall global rank (linked above analysis), an important distinction from individual score dropping or dropping in the Americas ranking. 

**SATISFACTORY:** Of the satisfactory nations, only Uruguay and Brazil had shown improvement. 5 countries reached "satisfactory" levels, with Canada and Jamacia shown improvement. 

As shown in the above chart (chart 2 - visual 3), Canada is the highest ranked country amongst those part of the Americas region. It is fairly in global rank (its 20th out of 180 countries - this is diff than general score - also just happened to be 20th in the list of countries so other numbers to left of countries above aren't their real global rank). Nicaragua is incredibly low in its global score at 24.98, placing it in RSF's "very serious" category, along with Cuba, Venezuela, Peru, and El Salvador. 9 of the countries are only a step abovethe "difficult" category and 9 are in the "problematic" category, leaving 5 in the "satisfactory" group, and none in the "good" category, which is the highest rank.

## **Step 8:** Find the average press freedom indicator scores across the Americas. This answers - **4) Which Press Freedom indicators are weakest across the region?/Which of RSF's five indicators has the lowest average score across the Americas?** This information will also be used for Chart 3.

In [37]:
indicator_cols = [
    "Political Context",
    "Economic Context",
    "Legal Context",
    "Social Context",
    "Safety",
]  

chart3_data = (
    Americas[indicator_cols]
    .mean()# average each column across all countries
    .round(2) #round the numbers by two decimal points like I did earlier to match.
    .reset_index()
)
chart3_data.columns = ["Indicator", "Average score"]
chart3_data = chart3_data.sort_values("Average score")

print(chart3_data)

           Indicator  Average score
1   Economic Context          43.91
0  Political Context          45.28
3     Social Context          59.50
2      Legal Context          60.25
4             Safety          67.80


### **Analysis 4:** 


According to RSF 2026 review(https://rsf.org/en/2026-rsf-index-press-freedom-25-year-low?data_type=general&year=2026), "the recent history of press freedom in the Americas has been shaped by the rise in violence perpetrated by two offenders: **organised crime and political actors**." In their text, they explain they use 5 indicators to measure press freedom: **political context, legal framework, economic context, sociocultural context and safety.**

**Political Actors:** Calls out Donald Trump specifically for using his power to influence press in the US and how political actors in foreign countries treat their media. It appears that the US has played a particularly significant role in press freedom in the Americas, with a hand in the decline of scores in individual countries, while losing score itself (7 points). It has also long been looked at as a leader in global press freedom and as a staunch supporter of the 1st amendment, but is now using its international influence to push for regression in press freedom. Specifically mentions Trump's supporters in Latin America (Argentina - difficult, and El Salvador -very serious)

"US President Donald Trump has turned his repeated attacks on the press and journalists into a systematic policy, pushing the US down to 64th place (-7). The detention of Salvadoran journalist Mario Guevara, who was later deported, has contributed to the deterioration of an already tense security environment marked by police violence. The drastic cuts to the US Agency for Global Media (USAGM) workforce had global repercussions, leading to the closure, suspension and downsizing of international broadcasters such as Voice of America (VOA), Radio Free Europe/Radio Liberty (RFE/RL) and Radio Free Asia (RFA) in countries where they were some of the last reliable sources of information."

"Presidents Javier Milei and Nayib Bukele —- Donald Trump’s most vocal supporters in Latin America — have taken their cue from the White House in their approach to the media, with unsurprisingly similar results. Javier Meili’s Argentina (98th, -11) and Nayib Bukele’s El Salvador (143rd, -8) have recorded significant declines, notably linked to the deterioration of these countries’ political and social indicators, reflecting a rise in government hostility towards and pressure on the press."

https://rsf.org/en/classement/2026/americas -> also explains how Trump's presidency has led to one of the most severe crises for press freedom in modern US history.

**Organized Crime: Links issue of organized crime to Ecuador, whose press freedom score, not global rank, dropped the most out of countries in the Americas since last year, along with Peru, which is also in the "very serious" category alongside Venezuela, Cuba and Nicaragua.**

"In countries where **organised crime** kills, Index rankings fall drastically. This is the case in Ecuador (125th), which has dropped 31 places since Darwin Baque and Patricio Aguilar were murdered in 2025. The same year, Peru (144th, -14) was impacted by the murder of four journalists. Guarantees for press freedom in Venezuela (159th) remain deeply uncertain despite the release of detained journalists earlier in the year. Finally, Cuba (160th) is undergoing a profound crisis that is forcing the few remaining independent journalists to operate underground, and Nicaragua’s (168th) media landscape lies in ruins, characterised by systematic repression and a lasting collapse in working conditions for journalists."

- **RSF 2026 Review ^^^**

**Indicators reveal the extent to which each of the 5 categories/indicators holds influence over its overall score, and in turn ranking beyond its region/at global level. When averaged out across countries in the Americas, the most dominant indicator proved to be economic context, with politics being a close second. In their review, RSF said legal context slipped the most across the world, which came in 2nd to last. 

**can find biggest indicators for countries in worst shape or decline most like i bet violence is highest in ecuador or something idk if time.**



## **CSV's Below**

In [28]:
region_scores_df.to_csv("RSF_region_scores_2026.csv", index=False)

In [30]:
map_data.to_csv("americas_press_freedom_change_2025_2026_v2.csv",index=False)

In [31]:
chart2_data.to_csv("americas_press_freedom_scores_2026.csv",index=False)

In [38]:
chart3_data.to_csv("americas_indicator_averages_2026.csv", index=False)